### ISSUE TITLE
Focus rings + keyboard navigation

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/SearchWorks#5627]
### From Darcy's review on 8/20/25
- [x] Skip to - Skip to search doesn't work (on home page and other pages) -- not sure where it's landing, but I can't seem to interact with it and there's no focus ring. Question though -- should it land on the input or the catalog/articles switch or the all fields dropdown? I lean toward input as a starting point.

- [ ] Header - Stanford University (a tiny bit more space around it)
- [ ] Header - No focus ring for Stanford University Libraries (focus happens, but you can't see it)
- [ ] Header/Search - Search mode - lands on first element in radio button group and allows you to move to the other option, but once pressed, you move to another page and lose the focus on the radio button (probably because it's different page, but wondering if we can make that experience more seamless)

- [x] Home - Featured resources -- focus ring should hug heading (instead of expanding to full width of heading)

- [ ] Search results - show/hide details -- focus ring should hug heading (same as above)

- [x] Search results - mini-bento - side of focus ring gets cut off by count (number of matches)
- [ ] Search results - search for a facet value - (minor) can the whole input (with the search icon) be part of the focus ring -- see screenshot for how it looks today.
  
<img width="396" height="130" alt="Image" src="https://github.com/user-attachments/assets/c81815fe-bd52-403b-b143-7a758a5115be" />
  
- [ ] Record view - tools (save, cite, email, etc.) - same focus ring (preferably the one used for cite, email, etc. is applied to save)
- [ ] Record view - Related items -- Needs discussion. Currently, it allows me to enter the area, scroll and select an item, but I only see a focus ring around the title. Can it also show the hover effect for the item too? Also, when I enter the scroll-able area for related items, could it scroll down to that place on the page? I entered it with it slightly hanging off the page. Also, should the user need to enter it with return or space? Similar to an accordion. Currently, you enter it with a tab, but you kind of end up trapped there. 
- [ ] Record view - Availability Expand - top of focus ring is cut-off

- [ ] Footer - No focus ring on Stanford University Libraries
- [ ] Footer - (minor) match the focus ring style for the SUL footer, it has a tiny bit more padding, the Stanford footer links don't

### Important 
[Defined focus ring style in Figma](https://www.figma.com/design/K1syTsalaB1nlRftgOi0Pw/DLSS-Components?node-id=802-1231&t=J5xViUNgKTk6ZGhF-4)

- 1px black ring around a 1px white ring (intended to work on both white or light backgrounds as well as black or dark backgrounds)
- Shown in Figma for buttons, but this is the intended style for all interactive elements on the page

**Comment by taylor-steve** [comment]
Some initial findings.

For buttons that are actually buttons (not links pretending to be buttons), the above linked style seems well implemented (2px black outline) via the component library.

Our links look like this:

<img width="543" height="42" alt="Image" src="https://github.com/user-attachments/assets/43f9b6b4-4ed2-486a-a5dd-5c995b9d4af1" />

Some potentially goofy looking things:

Links styled as buttons:
<img width="539" height="59" alt="Image" src="https://github.com/user-attachments/assets/27caea5a-201f-4382-8c43-27e5222db944" />
<img width="443" height="279" alt="Image" src="https://github.com/user-attachments/assets/e17e97ae-c91f-4d95-8dfd-f0c934e3e06a" />
<img width="275" height="60" alt="Image" src="https://github.com/user-attachments/assets/4a722500-2c47-4ac5-a51a-090c36512ecd" />


Not sure if a mix of shapes is bad?
<img width="78" height="52" alt="Image" src="https://github.com/user-attachments/assets/cbca831d-3e06-447f-b5d5-53c9f1aabb2a" />
<img width="91" height="52" alt="Image" src="https://github.com/user-attachments/assets/eaf24d16-fc05-459f-bb6c-3e85c81c8944" />
<img width="427" height="57" alt="Image" src="https://github.com/user-attachments/assets/3b37e8e5-0e17-477f-bd74-df6f06699343" />
<img width="439" height="57" alt="Image" src="https://github.com/user-attachments/assets/274df8ba-a95b-469f-8427-640a886b48fc" />
<img width="432" height="64" alt="Image" src="https://github.com/user-attachments/assets/4cef213f-8348-4b2e-bbf4-dad5597d89c4" />
<img width="431" height="60" alt="Image" src="https://github.com/user-attachments/assets/e5723f41-5fbb-48b9-a1ee-a66849b85a4a" />
<img width="428" height="63" alt="Image" src="https://github.com/user-attachments/assets/9af55cda-6f28-4750-a661-474fae4a6285" />


Wrapped links look awkward to me:
<img width="156" height="129" alt="Image" src="https://github.com/user-attachments/assets/8eda4523-124a-4932-99f2-a4264dbdd4e1" />
<img width="602" height="339" alt="Image" src="https://github.com/user-attachments/assets/e884dd91-5ca0-4d09-848d-f52db57161f8" />

**Comment by dbranchini** [comment]
@taylor-steve, the double-outline is only so that we don't need to mess with focus rings for different background (dark/black or light/white). If we can't, then I vote for a simple black outline since it should work for all of SearchWorks.

**Comment by taylor-steve** [comment]
@dbranchini regarding our discussion earlier today:

* I was wrong, Earthworks *is* using the component library outline + box-shadow for link focus-active.
* The difference is that they are using `display: block` for those record links, that's why the box shadow is a block rather than the wrapping behavior we see in SearchWorks.

I think this puts us back to where we were, needing SearchWorks specific styles or completely rethinking what the component library is doing as we can't switch to block styling for those links.
